sktime/transformations/series/summarize.py

```diff
@@ -289,9 +289,15 @@ def _transform(self, X, y=None):
         return Xt_return
 
     @classmethod
-    def get_test_params(cls):
+    def get_test_params(cls, parameter_set="default"):
         """Return testing parameter settings for the estimator.
 
+        Parameters
+        ----------
+        parameter_set : str, default="default"
+            Name of the set of test parameters to return. The method must be overridden
```

Comment:
the docstring is not clear. What does this mean "must be overridden"?

Suggest the following string instead
```Name of the set of test parameters to return, for use in tests. If no special parameters are defined for a string, will always return the `"default"` set.```

Reply:
Done, but may change again. See part 2 of below comment.